addon/src/main/java/com/vaadin/addon/charts/model/Configuration.java

```diff
@@ -1140,4 +1157,17 @@ public void addColorAxis(ColorAxis axis) {
         colorAxis.addAxis(axis);
     }
 
+    /*
+     * Helper function for conent sanitization, this preserves common formmatting, but
+     * strips scripts.
+     */
+    String sanitize(String html) {
+        return org.jsoup.Jsoup.clean(html,
+                org.jsoup.safety.Whitelist.basic()
+                        .addTags("img", "h1", "h2", "h3", "s")
+                        .addAttributes("img", "align", "alt", "height", "src",
+                                "title", "width")
+                        .addAttributes(":all", "style")
+                        .addProtocols("img", "src", "data"));
```

Comment:
What's the basis for this specific set of allowed values?

Reply:
This is the same set we use in Vaadin 14 version of RichTextEditor for the same purpose

Reply:
Don't tell it to me, tell it ot the next person who reads the code and wonders what the reasoning is behind this specific set.